app/routes.py

```diff
@@ -0,0 +1,357 @@
+from app import db
+from flask import Blueprint, jsonify, request
+from datetime import datetime, timedelta
+from app.models.video import Video
+from app.models.customer import Customer
+from app.models.rental import Rental
+
+
+videos_bp = Blueprint("videos", __name__, url_prefix=("/videos"))
+
+#POST/CREATE 
+@videos_bp.route("",methods=["POST"])
+def post_videos():
+    request_body = request.get_json()
+    if "title" not in request_body:
+        return{
+            "details": "Request body must include title."
+        }, 400
+    
+    elif "release_date" not in request_body:
+        return{
+            "details": "Request body must include release_date."
+        }, 400
+    
+    elif "total_inventory" not in request_body:
+        return{
+            "details": "Request body must include total_inventory."
+        }, 400
+
+    new_video = Video(
+        title = request_body["title"], 
+        release_date = request_body["release_date"],
+        total_inventory = request_body["total_inventory"]
+    )
+    db.session.add(new_video)
+    db.session.commit()
+
+    return jsonify(new_video.to_video_object()), 201
+
+#GET
+@videos_bp.route("", methods=["GET"])
+def get_videos():
+    videos = Video.query.all()
+    videos_response = []
+    for video in videos:
+        videos_response.append(video.to_video_object())
+    return jsonify(videos_response), 200
+
+@videos_bp.route("/<video_id>",methods=["GET"])
+def get_video(video_id):
+    if video_id.isnumeric() != True:
```

Comment:
The most dependable way to check whether a python string represents an integral value is to try to convert it to an int in a try block, then if it raises a ValueError, we know it wasn't an int, so here, that could look like:

```python
    try:
        int(video_id)
    except ValueError:
        return {"message": "Video id provided is not a number."}, 400 
```

The string methods like `isnumeric` and `isdigit` are more about detecting classes of characters (think Unicode) than they are for determining what numeric types we might be able to convert the string value into.